gems/aws-sdk-s3/spec/presigner_spec.rb

```diff
@@ -75,19 +75,22 @@ module S3
             bucket: 'examplebucket',
             key: 'test.txt',
             expires_in: 86_400,
-            time: Time.utc(1969, 4, 20)
```

Comment:
!

Reply:
That actually does not work in our CRT FFI bindings (dates pre-1970 will not be represented correctly).